# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../konversation/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur konversation-0.19-orig/konversation/src/searchbar.h konversation-0.19/konversation/src/searchbar.h
--- konversation-0.19-orig/konversation/src/searchbar.h	2006-01-30 08:43:47.000000000 +0000
+++ konversation-0.19/konversation/src/searchbar.h	2006-04-04 14:10:04.537269500 +0000
@@ -55,7 +55,7 @@
 
         void focusInEvent(QFocusEvent* e);
         virtual void showEvent(QShowEvent* e);
-        bool SearchBar::focusedChild();
+        bool focusedChild();
     public slots:
 
         virtual void hide();
